tests/pipeline/embed/test_omnibus_embedding.py

```diff
@@ -154,13 +154,14 @@ def test_omnibus_embedding_elbowcuts_none_returns_full_embedding(self):
     def test_omnibus_embedding_digraph_elbowcuts_none_returns_full_embedding(self):
         dimensions = 100
         expected_dimensions = dimensions * 2
+        number_of_nodes = 1000
```

Comment:
I tested this for  2x100k nodes - previously required 74.5gb for numpy to allocate the adj matrix, now completes in a little under a minute on my machine.

Reply:
What is runtime for numpy arrays? What is the memory requirement with sparse?

Reply:
Runtime for numpy for 100k is a gigantic explosion of doom as I don't have 75 GB of ram. When run on a small dataset of ~1k nodes in each graph, sparse takes 2.85 seconds and dense takes 3.3 seconds.

It is going to be highly dependent on the sparsity of your graph, of course.